spec/features/login_spec.rb

```diff
@@ -1,33 +1,22 @@
 require 'rails_helper'
+require 'support/sign_in_helper'
+require 'support/sign_out_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.